setup.cfg

```diff
@@ -35,128 +35,6 @@ per-file-ignores =
 # B950 will only trigger at 88
 max-line-length = 80
```

Comment:
Wonder if we should add issue ( https://github.com/PyCQA/flake8/issues/234 ) to the references above to note `pyproject.toml` is not yet supported by `flake8`